src/components/Shared/LinePlot/LinePlotComponent.tsx

```diff
@@ -122,6 +122,7 @@ export class LinePlotComponentProps {
     insideTexts?: LinePlotInsideTextMarker[];
     order?: number;
     multiPlotPropsMap?: Map<string, MultiPlotProps>;
+    exportData?: {x: number; y: number; z?: number}[];
```

Comment:
Maybe not related to the fix, but I wonder if we can remove `z` and use `Point2D[]` for `exportData` and also `data`. Seems that `z` is not used, and it was for inheritance in `ScatterPlot`.